appservice/src/index.ts

```diff
@@ -17,23 +14,27 @@ export * from './createAppService/AppServicePlanSkuStep';
 export * from './createAppService/CustomLocationListStep';
 export * from './createAppService/IAppServiceWizardContext';
 export * from './createAppService/LogAnalyticsCreateStep';
+export * from './createAppService/setLocationsTask';
+export * from './createAppService/SiteDomainNameLabelScopeStep';
```

Comment:
Don't forget to add this to the index.d.ts or else you won't be able to actually import it in the extension without typings yelling at you.

Reply:
This is one of our projects that utilize the automatically generated declarations:
![image](https://github.com/user-attachments/assets/24eb5b18-6a34-46a0-adce-3f4c0edc3b2d)

Confusing sometimes because not all our older projects do it that way :P